src/frontend/src/views/Upload/components/Samples/strings.ts

```diff
@@ -0,0 +1,14 @@
+import { StringsType } from "src/common/types/strings";
+
+interface SamplesStrings {
```

Comment:
Nit: I think this would be better as `UploadSamplesStrings`. To me, these strings are as much about the upload aspect as the samples aspect.

Reply:
I used the component name, but I agree - especially since we have a nearly identical UI but different codepath for the edit flow.